qa/L0_trt_plugin/test.sh

```diff
@@ -135,7 +135,7 @@ SERVER_LD_PRELOAD=$CUSTOMPLUGIN
 SERVER_ARGS=$SERVER_ARGS_BASE
 SERVER_LOG="./inference_server_$LOG_IDX.log"
 
-if  [[ ! -v WSL_DISTRO_NAME ]] || [[ ! -v MSYSTEM ]]; then
+if  [[ ! -v WSL_DISTRO_NAME ]] && [[ ! -v MSYSTEM ]]; then
```

Comment:
Not familiar with MSYSTEM - but what's the scenario where one is set and not the other?

Reply:
Variable `WSL_DISTRO_NAME` applicable to WSL environment only. <br>
Variable `MSYSTEM` is applicable for GNU on Windows. 

Afraid condition will be always `false`

Reply:
My understanding is that we use `MINGW64` as our shell interpreter for Windows, which _should_ always set `MYSYSTEM`. In fact, if you examine our runners, `MYSYSTEM` has a value, however, `WSL_DISTRO_NAME` does not. I'm not sure of a scenario where the opposite would be true and `WSL_DISTRO_NAME` has a value but `MSYSTEM` does not. Perhaps @mc-nv knows?

Reply:
The original intention of this if-condition was to skip the enclosed test on a Windows system. On a Linux system, neither variable will be set, so the enclosed test will be run. On a Windows system, we anticipate that at least one of these will be set, ensuring that we skip the test.

Reply:
nit: In that case, can we either:

(a) match the if statement at the top for consistency?
```bash
if [[ -v WSL_DISTRO_NAME ]] || [[ -v MSYSTEM ]]; then
  echo "Skip this section on Windows"
else
  ...
fi
```

(b) or set some syntactic sugar naming ex:
```bash
IS_WINDOWS=...

if [ IS_WINDOWS ] 
# or
if [ ! IS_WINDOWS ]
```